manager.py

```diff
@@ -0,0 +1,160 @@
+import docker
+import psycopg2
+import json
+import time
+
+
+def get_env_vars(client):
```

Comment:
I meant the environment variables should be supplied to _this_ container, not that it should fetch them from the master. I'll update this, too.

Reply:
OK, so yeah, these variables should be passable to _this_ script, and it should honor them. So I guess modify it to permit that.

Reply:
When I check the docker/develop branch, I see that you removed the environment variables from the compose file. Which is appropriate when we consider your first comment. However, I don't understand how you want us to provide these variables. If we do it from docker-compose.yml/manager with the "environment:" keyword, the same way as we do for the master, it is straightforward to implement. However, if we put these variables into manager without master knowing, we also need to create the desired ROLE, DATABASE and PASSWORD in master, remotely. I don't think, this is also what you wanted.